ballerina/Dependencies.toml

```diff
@@ -0,0 +1,341 @@
+# AUTO-GENERATED FILE. DO NOT MODIFY.
```

Comment:
Do need to commit Dependencies.toml?

Reply:
+1 to commit the Dependencies.toml, which is the common convention for the other extended libraries as well